locales/de.json

```diff
@@ -1,574 +1,911 @@
+      "description": "",
+      "help": "Der Wert des id-Attributes muss einzigartig sein."
+    },
+    "empty-heading": {
+      "description": "",
+      "help": "Überschriften dürfen nicht leer sein."
+    },
+    "focus-order-semantics": {
+      "description": "",
+      "help": "Elemente in der Fokusreihenfolge benötigen eine Rolle, die für interaktive Elemente geeignet ist."
+    },
+    "form-field-multiple-labels": {
+      "description": "Stellt sicher, dass ein form-Feld nur ein label-Element besitzt.",
+      "help": "form-Felder sollten nur ein label-Element besitzen."
+    },
+    "frame-tested": {
+      "description": "",
+      "help": "Frames müssen mit axe-core getestet werden."
+    },
+    "frame-title-unique": {
+      "description": "",
+      "help": "Frames müssen ein einzigartiges title-Attribut besitzen."
+    },
+    "frame-title": {
+      "description": "",
+      "help": "Frames müssen ein title-Attribut besitzen."
+    },
+    "heading-order": {
+      "description": "",
+      "help": "Überschriftenebenen sollten nur jeweils um eins steigen."
+    },
+    "hidden-content": {
+      "description": "",
+      "help": "Versteckter Inhalt auf der Seite konnte nicht analysiert werden."
+    },
+    "html-has-lang": {
+      "description": "",
+      "help": "Das <html>-Element muss ein lang-Attribut besitzen."
+    },
+    "html-lang-valid": {
+      "description": "",
+      "help": "Das <html>-Element muss einen gültigen Attributwert für das lang-Attribut besitzen."
+    },
+    "html-xml-lang-mismatch": {
+      "description": "Stellt sicher, dass HTML Elemente mit validen lang und xml:lang Attributen diesselbe Angabe über die Sprache machen.",
+      "help": "HTML Elemente mit lang und xml:lang Attributen müssen diesselbe Sprache ausweisen."
+    },
+    "identical-links-same-purpose": {
+      "description": "Stellt sicher, dass links mit dem selben Namen (accessible-name) denselben Zweck folgen.",
+      "help": "Links mit dem selben Namen verfolgen denselben Zweck."
+    },
+    "image-alt": {
+      "description": "",
+      "help": "Abbildungen müssen einen Alternativtext besitzen."
+    },
+    "image-redundant-alt": {
+      "description": "",
+      "help": "Der Text von Schaltflächen oder Links sollte in der Abbildungsalternative nicht wiederholt werden."
+    },
+    "input-button-name": {
+      "description": "Stellt sicher, dass Eingabeschaltflächen sichtbaren Text beinhalten.",
+      "help": "Eingabeschaltflächen müssen sichtbaren Text beinhalten."
+    },
+    "input-image-alt": {
+      "description": "",
+      "help": "Bild-Schaltflächen müssen einen Alternativtext besitzen."
+    },
+    "label-content-name-mismatch": {
+      "description": "Stellt sicher, dass Elemente die durch ihren Inhalt beschrieben sind auch ihren sichtbaren Text als Teil des zugänglichen Namens haben.",
+      "help": "Elemente müssen ihren sichtbaren Text auch als Teil des zugänglichen Namens haben."
```

Comment:
I prefer `Zugänglichen Namen` rather than `Barrierfreien Namen` - but either way we should be consistent. WDYT?

Reply:
Very good question. I found 2.5.3 in a translation also as `Beschriftung (Label) im Namen` or `Label in Name`. Don't know what would be the best for users without certain knowledge.

Reply:
Iwas actually referring to the terminiligy used throughout the translation. Sometimes we use `Zugänglich` and sometimes `Baririerfrei` and I think we should pick one. Which do you prefer?

Reply:
I meant that it could be, that it's not really clear to the user what `zugänglicher Name` means. But I prefer `zugänglich` over `barrierefrei`.

Reply:
Nothing I can solve, maybe there will be a better "official" translation in the future…